configs/components/curl.rb

```diff
@@ -0,0 +1,45 @@
+component 'curl' do |pkg, settings, platform|
+  pkg.version '7.46.0'
+  pkg.md5sum '230e682d59bf8ab6eca36da1d39ebd75'
+  pkg.url "http://buildsources.delivery.puppetlabs.net/curl-#{pkg.get_version}.tar.gz"
```

Comment:
It's a mirror of https://curl.haxx.se/download/curl-7.46.0.tar.gz. We should be using Vanagon's URL rewriting to point to that mirror, not putting it in components. Not sure when I'll get around to fixing that however.